apps/backend/src/libs/modules/server-application/base-server-application.ts

```diff
@@ -166,16 +166,19 @@ class BaseServerApplication implements ServerApplication {
 	}
 
 	public addRoute(parameters: ServerApplicationRouteParameters): void {
-		const { handler, method, path, validation } = parameters;
+		const { handler, method, path, preHandlers, validation } = parameters;
 
-		this.app.route({
+		const routeOptions = {
 			handler,
 			method,
+			preHandlers: preHandlers ?? [],
```

Comment:
I think fastify accepts preHandler only property, preHandlers is like our internal name, because we know it will be always an array
```suggestion
			preHandler: preHandlers ?? [],
```